packages/common/src/utils/sagaHelpers.ts

```diff
@@ -166,6 +166,7 @@ export function* sagaWithErrorHandler(saga: () => Generator<any, void, any>) {
         yield* getContext('reportToSentry') as any
 
       yield* call(reportToSentry, {
+        name: 'Uncaught Saga Error',
```

Comment:
This is the global saga handler - adding a name to better bucket these errors

Reply:
TODO: should I tag this? What would be appropriate?

Reply:
Hmm I think the title is good enough for single error type buckets

We can use tags for buckets that log in several different places and use titles to specify within each.